arrow-core-data/src/main/kotlin/arrow/typeclasses/Foldable.kt

```diff
@@ -205,13 +218,27 @@ interface Foldable<F> {
   /**
    * Get the first element of the foldable or none
    */
+  @Deprecated("In favor of having a more Kotlin idiomatic API", ReplaceWith("first()"))
   fun <A> Kind<F, A>.firstOption(): Option<A> =
-    find { true }
+    first()
 
   /**
    * Get the first element matching the predicate or none
    */
+  @Deprecated("In favor of having a more Kotlin idiomatic API", ReplaceWith("firstOrNone(predicate)"))
   fun <A> Kind<F, A>.firstOption(predicate: (A) -> Boolean): Option<A> =
+    firstOrNone { predicate(it) }
+
+  /**
+   * Get the first element of the foldable or none
+   */
+  fun <A> Kind<F, A>.first(): Option<A> =
```

Comment:
imo this should be `firstOrNone` and not `first`

Reply:
Changed it! thanks! 👍